### PR TITLE
Update Redis integration tests to use ConsoleMF

### PR DESCRIPTION
**Pull request by chynesNR**
* Converted StackExchange.Redis tests to use ConsoleMF
* Test covers latest version plus 5 others (all passing)
* This also allows us to remove the separate StackExchange.Redis.StrongName test classes
* Turned on Multiverse Scanner for Redis

### CODE REVIEW

File: tests/Agent/MultiverseTesting/ConsoleScanner/config.yml [hunk 1]
```diff
@@ -100,12 +100,12 @@
...
-  #   xml-file: ${{ MVS_XML_PATH }}/StackExchangeRedis/Instrumentation.xml
-  #   nuget-packages:
-  #      - { package-name: StackExchange.Redis, versions: [1.0.488, 1.1.608, 1.2.6, 2.0.601, 2.2.4] }
-  #      - { package-name: StackExchange.Redis.StrongName, versions: [1.0.488, 1.1.608, 1.2.6] }
-  #   local-assemblies:
+  - name: stackExchangeRedis
+    xml-file: ${{ MVS_XML_PATH }}/StackExchangeRedis/Instrumentation.xml
+    nuget-packages:
+       - { package-name: StackExchange.Redis, versions: [1.0.488, 1.1.608, 1.2.6, 2.0.601, 2.2.4] }
+       - { package-name: StackExchange.Redis.StrongName, versions: [1.0.488, 1.1.608, 1.2.6] }
```
Comment (JcolemanNR) [inline tests/Agent/MultiverseTesting/ConsoleScanner/config.yml:107]: I guess I would expect these versions sync up with, or at least include the versions that we are testing in our integration tests.

File: tests/Agent/IntegrationTests/UnboundedIntegrationTests/Redis/StackExchangeRedisTests.cs [hunk 2]
```diff
@@ -11,50 +10,56 @@
...
-        public StackExchangeRedisTests(RemoteServiceFixtures.BasicMvcApplication fixture, ITestOutputHelper output)  : base(fixture)
+        public StackExchangeRedisTestsBase(TFixture fixture, ITestOutputHelper output) : base(fixture)
         {
             _fixture = fixture;
             _fixture.TestLogger = output;
+
+            _fixture.AddCommand($"StackExchangeRedisExerciser DoSomeWork");
+            _fixture.AddCommand($"StackExchangeRedisExerciser DoSomeWorkAsync");
+
             _fixture.Actions
```
Comment (JcolemanNR) [inline tests/Agent/IntegrationTests/UnboundedIntegrationTests/Redis/StackExchangeRedisTests.cs:29]: You could call `_fixture.AddActions` here and include your wait for log lines logic.

File: tests/Agent/IntegrationTests/SharedApplications/Common/MultiFunctionApplicationHelpers/MultiFunctionApplicationHelpers.csproj [hunk 3]
```diff
@@ -50,6 +50,16 @@
     <PackageReference Include="MySqlConnector" Version="1.0.1" Condition="'$(TargetFramework)' == 'netcoreapp3.1'" />
     <PackageReference Include="MySqlConnector" Version="1.3.13" Condition="'$(TargetFramework)' == 'net5.0'" />
     <PackageReference Include="MySqlConnector" Version="2.1.2" Condition="'$(TargetFramework)' == 'net6.0'" />
+
+    <!-- StackExchange.Redis framework references -->
+    <PackageReference Include="StackExchange.Redis.StrongName" Version="1.1.608" Condition="'$(TargetFramework)' == 'net462'" />
+    <PackageReference Include="StackExchange.Redis" Version="2.0.601" Condition="'$(TargetFramework)' == 'net471'" />
+    <PackageReference Include="StackExchange.Redis" Version="2.2.88" Condition="'$(TargetFramework)' == 'net48'" />
```
Comment (JcolemanNR) [inline tests/Agent/IntegrationTests/SharedApplications/Common/MultiFunctionApplicationHelpers/MultiFunctionApplicationHelpers.csproj:57]: I kind of think we should test the Min/Max supported versions for both framework and core

**Review by JcolemanNR — APPROVED** [review]
Looks good! I have a few suggestions, but nothing I feel strongly enough to 'request changes' for

File: tests/Agent/IntegrationTests/UnboundedIntegrationTests/Redis/StackExchangeRedisTests.cs (hunk 2 above)
Comment (chynesNR) [reply tests/Agent/IntegrationTests/UnboundedIntegrationTests/Redis/StackExchangeRedisTests.cs:29]: Oh I see what you mean. I agree, that's a little cleaner. Done.

File: tests/Agent/MultiverseTesting/ConsoleScanner/config.yml (hunk 1 above)
Comment (chynesNR) [reply tests/Agent/MultiverseTesting/ConsoleScanner/config.yml:107]: You caught me. I didn't notice the discrepancy until after I pushed. :)

File: tests/Agent/IntegrationTests/SharedApplications/Common/MultiFunctionApplicationHelpers/MultiFunctionApplicationHelpers.csproj (hunk 3 above)
Comment (JcolemanNR) [reply tests/Agent/IntegrationTests/SharedApplications/Common/MultiFunctionApplicationHelpers/MultiFunctionApplicationHelpers.csproj:57]: I think it depends on the library as some have different versions supported across framework/core. I see your logic here and agree that maximizing our test surface is a good idea.

**Pull request merged**